firmware/integration/rusefi_config.txt

```diff
@@ -155,6 +155,9 @@ struct_no_prefix engine_configuration_s
 #define BARO_CORR_SIZE 4
 
 #define MAF_DECODING_COUNT 256
+#define AFTERSTART_HOLD_CURVE_SIZE 8
```

Comment:
do we actually need this much resolution? Is 6 enough?

Reply:
Probably not needed.
Just used to see all these curves with 8 :-)